Extractor/Nodes/BaseUANode.cs

```diff
@@ -562,6 +563,84 @@ protected Dictionary<string, string> BuildMetadataBase(Dictionary<string, string
             return result;
         }
 
+        protected Dictionary<string, JsonNode?> BuildMetadataJsonBase(Dictionary<string, JsonNode?>? extras, IUAClientAccess client, bool reversibleJson = true)
+        {
+            var result = extras ?? new Dictionary<string, JsonNode?>();
+
+            if (Properties == null) return result;
```

Comment:
I think "return result" should be on a separate line.
This is handy for debugging this part of the code, by setting a breakpoint on that separate line.

But this is perhaps not what is enforced by the current .editorconfig?

Reply:
No. I can make the `return` on a separate line.

Reply:
Sorry to have to come in and disagree with the risk reviewer here, but in this codebase we overwhelmingly use same-line return. While @msusag has a fair point, I personally really dislike implicit blocks, they are dangerous, so this would have to become

```suggestion
            if (Properties == null)
            {
                return result;
            }
```

That said, I think that consistency trumps any specific style. I would rather have a consistently mediocre style, than an _inconsistent_ style. So please keep it same-line for now.

Reply:
Then we agree to disagree. Fair enough.
This is a code style thing which is subjective, where different teams have different preferences.

But this code base will then use a different code standard than in other Cognite C# repos and other .NET repos that follow the dotnet runtime editorconfig.